### PR TITLE
Daniel's homework week-1

### PR DESCRIPTION
**Pull request by DanGhanbari**

### CODE REVIEW

File: week-1-practise/level-1/10-function-output-3.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 function concatenate(firstWord, secondWord, thirdWord) {
     // Write the body of this function to concatenate three words together
+    return firstWord.concat(secondWord).concat(thirdWord)
```
Comment (amaalali) [inline week-1-practise/level-1/10-function-output-3.js]: Good stuff. You can also make things easier by passing multiple arguments to `concat`. Take a look a the docs here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#Syntax.

As an interesting note, there are multiple ways to do everything, and you also use the `+` to achieve the same result and there are slight differences in the way they work https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#Performance.

File: week-1-practise/level-1/7-logic-error-4.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function multiply(a, b, c) {
-  a * b * c;
-  return;
+  
+  return a * b * c;
```
Comment (amaalali) [inline week-1-practise/level-1/7-logic-error-4.js]: :+1:

File: week-1-practise/level-1/7-logic-error-4.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 }
 
-var result = multiply(1, 3, 4);
+const result = multiply(1, 3, 4);
```
Comment (amaalali) [inline week-1-practise/level-1/7-logic-error-4.js]: Nice one to recognize that this can be a `const`

File: week-1-practise/level-3/1-piping.js [hunk 4]
```diff
@@ -11,6 +11,7 @@
 */
 
 var startingValue = 5;
+ result = (format( add (multiply (5,2),10)));
```
Comment (amaalali) [inline week-1-practise/level-3/1-piping.js]: We don't normally leave spaces between the function name and the parameter list

```suggestion
 result = format( add( multiply(5,2), 10));
```

File: week-1-practise/level-1/3-syntax-error-3.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 // There is a syntax error with this code. Fix it.
 
-function(a, b, c) {
+function addNumbers(a, b, c) {
```
Comment (amaalali) [inline week-1-practise/level-1/3-syntax-error-3.js]: Interestingly there is another way to solve this, search for Javascript and anonymous functions

File: week-1-practise/level-1/5-logic-error-2.js [hunk 6]
```diff
@@ -1,7 +1,7 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return word.trim;
+  return word.trim();
```
Comment (amaalali) [inline week-1-practise/level-1/5-logic-error-2.js]: Correct! Did you take note of what `word.trim` returned?

Comment (DanGhanbari) [reply week-1-practise/level-1/5-logic-error-2.js]: yes. it removed unnecessary spaces

File: week-1-practise/level-1/7-logic-error-4.js (hunk 2 above)
Comment (DanGhanbari) [reply week-1-practise/level-1/7-logic-error-4.js]: 😊

File: week-1-practise/level-1/7-logic-error-4.js (hunk 3 above)
Comment (DanGhanbari) [reply week-1-practise/level-1/7-logic-error-4.js]: 👍

File: week-1-practise/level-1/10-function-output-3.js (hunk 1 above)
Comment (DanGhanbari) [reply week-1-practise/level-1/10-function-output-3.js]: Do you mean in this way:
function concatenate(firstWord, secondWord, thirdWord) {
return firstWord + secondWord + thirdWord;
}

**Pull request closed without merging**